include/z64audio.h

```diff
@@ -16,7 +16,20 @@
 #define ADSR_GOTO -2
 #define ADSR_RESTART -3
 
-#define AIBUF_LEN 0x580
+// size of a single sample point
+#define SAMPLE_SIZE sizeof(s16)
+
+// Samples are processed in groups of 16 called a "frame"
+#define SAMPLES_PER_FRAME ADPCMFSIZE
+
+// The size of one left/right channel is 13 frames
```

Comment:
"the size is 13 frames" ? I'm afraid I don't get it

Reply:
As in, "1 channel is 0x1A0 bytes long, which is 0xD0 samples long, which is 0xD frames long"

Reply:
Switched to a "length" of 13 frames, that's probably clearer?

Reply:
Yeah that's better. (I expected size == bytes)